man/corosync.conf.5

```diff
@@ -273,8 +273,15 @@ is 8. For other transports it is 1.
 
 .TP
 netmtu
-This specifies the network maximum transmit unit.  To set this value beyond
-1500, the regular frame MTU, requires ethernet devices that support large, or
+This specifies maximum packet length sent by corosync. It's mainly for UDPU
+(and UDP) transport, where it specifies the network maximum transmit unit, but
+can be used also with KNET transport, where it defines maximum length of packet
+passed to knet layer. To specify network MTU manually for KNET, use
+.B knet_mtu
+option.
+
+For UDPU (and UDP) to set this value beyond 1500, the regular frame MTU,
+requires ethernet devices that support large, or
```

Comment:
"This specifies maximum packet length sent by corosync. It's mainly for the UDPU
(and UDP) transport, where it specifies the network maximum transmit size, but
can be used also with the KNET transport, where it defines the maximum length of packets
passed to the knet layer. To specify the network MTU manually for KNET, use the
.B knet_mtu
option.

For UDPU (and UDP), setting this value beyond 1500, the regular frame MTU,
requires ethernet devices that support large, or"